.github/workflows/test.yml

```diff
@@ -57,8 +57,6 @@ jobs:
             otp-version: '22'
           - elixir-version: 'master'
             otp-version: '^22'
-          - elixir-version: 'master'
```

Comment:
This was repeated two combos above. Also, I'm not sure all these combos are actually required (though they did help me during dev.)